queries/rmrk/subsquid/collectionEntityById.graphql

```diff
@@ -0,0 +1,22 @@
+query collectionEntityById($id: ID!, $nftId: ID!) {
+  collectionEntityById(id: $id) {
+    id
+    name
+    nfts(
+      limit: 10
+      orderBy: [price_ASC, updatedAt_DESC]
+      where: { price_gt: "0", id_not_eq: $nftId }
+    ) {
+      name
+      price
+      id
+      currentOwner
+      issuer
+      meta {
+        id
+        image
+      }
+      updatedAt
```

Comment:
make a fragment please so we do not copy paste the variables

```graphql
fragment nft on NFTEntity {
  name
  price
  id
  currentOwner
  issuer
}

fragment baseMeta on Metadata {
  id
  image
}
```

Reply:
hi @vikiival, need your help with this
1. I tried to use this one https://github.com/kodadot/nft-gallery/blob/main/queries/fragments/nft.graphql got the error ```Cannot query field \"collectionId\" on type \"NFTEntity\". Did you mean \"collection\"?```
2. created `baseMeta` got this error `Unknown type \"Metadata\".`

did I miss something?
my updated query:
```graphql
#import "../../fragments/nft.graphql"
#import "../../fragments/nftDetails.graphql"
#import "../../fragments/baseMeta.graphql"

query collectionEntityById($id: ID!, $nftId: ID!) {
  collectionEntityById(id: $id) {
    id
    name
    nfts(
      limit: 10
      orderBy: [price_ASC, updatedAt_DESC]
      where: { price_gt: "0", id_not_eq: $nftId }
    ) {
      ...nftDetails
      meta {
        ...baseMeta
      }
      updatedAt
    }
  }
}

```

Reply:
Yup my bad
```graphql
fragment baseMeta on MetadataEntity {
  id
  image
}
```

Reply:
hi @vikiival, how about the 1st one. should I refactor the others or leave at it is

Reply:
> how about the 1st one 

I missed that. Let's create a new nftdetail for subsquid and we will refactor that later 💯